index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Elson Otake</title>
+</head>
+<body>
+  <header>
+    <ul id="header_list">
+      <li id="company">Origami Inc</li>
+      <li id="menu">
+          <span class="material-symbols-rounded" id="hamburger">menu</span>
+      </li>
+    </ul>
+  </header>
+  <div id="presentation">
+    <div id="pres">
```

Comment:
A container tag of a section should always be a section tag. kindly change div tag in line 22 to section tag. 
[CAUTION] IDs have a much higher specificity than classes. A class on its own can't overwrite styles belonging to an ID. To  beat the ID, you would need either more IDs or to use ! important in your css which can also lead to specifity wars in your stylesheets. My advise is kindly use classes not IDs to avoid specifity problems. 😃